Airports/EGNS/Positions.txt

```diff
@@ -1,4 +1,4 @@
-EGNS_A_APP:Ronaldsway Approach:135.900:NSA:S:EGNS:APP:-:-:4550:4567:N054.05.00.000:W004.37.26.000::
+EGNS_A_APP:Ronaldsway Approach:135.905:NSA:S:EGNS:APP:-:-:4550:4567:N054.05.00.000:W004.37.26.000::
 EGNS_APP:Ronaldsway Radar:120.850:NSR:S:EGNS:APP:-:-:4550:4567:N054.05.00.000:W004.37.26.000::
 EGNS_F_APP:Ronaldsway Director:125.300:NSF:S:EGNS:APP:-:-:4550:4567:N054.05.00.000:W004.37.26.000::
 EGNS_TWR:Ronaldsway Tower:119.000:NST:S:EGNS:TWR:-:-:4550:4567:N054.05.00.000:W004.37.26.000::
```

Comment:
```suggestion
EGNS_APP:Ronaldsway Radar:120.855:NSR:S:EGNS:APP:-:-:4550:4567:N054.05.00.000:W004.37.26.000::
EGNS_F_APP:Ronaldsway Director:125.305:NSF:S:EGNS:APP:-:-:4550:4567:N054.05.00.000:W004.37.26.000::
EGNS_TWR:Ronaldsway Tower:119.005:NST:S:EGNS:TWR:-:-:4550:4567:N054.05.00.000:W004.37.26.000::
```
ATIS also needs changing but won't let me suggest on it, should be 123.880, [see AIP](https://www.aurora.nats.co.uk/htmlAIP/Publications/2023-11-02-AIRAC/html/eAIP/EG-AD-2.EGNS-en-GB.html#EGNS-AD-2.18)

Reply:
Positions updated